set.mm

```diff
@@ -465412,8 +465454,8 @@ its graph has a given second element (that is, function value).
     fnimatp.2 $e |- ( ph -> A e. D ) $.
     fnimatp.3 $e |- ( ph -> B e. D ) $.
     fnimatp.4 $e |- ( ph -> C e. D ) $.
-    $( The image of a triplet under a function.  (Contributed by Thierry
-       Arnoux, 19-Sep-2023.) $)
+    $( The image of a triple under a function.  (Contributed by Thierry Arnoux,
```

Comment:
above, "unordered triple" is used
additionally above, "unordered pair" is changed to simply "pair"

personally I prefer concision but changing this (and again once below) to "unordered triplet" is ok too

Reply:
Thanks. Added.